java/src/main/java/com/powsybl/python/SeriesPointerArrayBuilder.java

```diff
@@ -250,4 +289,15 @@ public PointerBase createDataPtr(List<T> elements) {
 
         return PyPowsyblApiHeader.allocArrayPointer(seriesPtr, seriesList.size());
     }
+
+    // used for unittest
+    List<List> buildJavaSeries() {
```

Comment:
Looks good : actually maybe we can just return the `List<Series>` ? this way we can check also the series names.